polyfill/lib/calendar.mjs

```diff
@@ -1322,6 +1322,14 @@ const helperIslamic = ObjectAssign({}, nonIsoHelperBase, {
   DAYS_PER_ISLAMIC_YEAR: 354 + 11 / 30,
   DAYS_PER_ISO_YEAR: 365.2425,
   constantEra: 'ah',
+  reviseIntlEra(calendarDate /*, isoDate*/) {
+    let { era, eraYear } = calendarDate;
+    // Chrome for Android as of v 142.0.6367.179 mishandled the era option in Intl.DateTimeFormat
+    // and returned 'bc' instead of 'ah'. This code corrects that.
+    // see https://issues.chromium.org/issues/40856332
+    if (era === 'before-christ' || era === 'bc' || era === 'b') era = 'ah';
```

Comment:
Since the era is always expected to be `AH` I am starting to think that the condition might not be necessary and we can simply override the era directly `era = 'ah'` and return it.

Let me know if that is a better patch then what we have here and I can change it.

Reply:
Yep, just unconditionally setting it to `'ah'` seems like the easiest (and smallest bundle size) solution.

Reply:
Done